### PR TITLE
Disable map

### PR DESCRIPTION
**Pull request by Giggzzz**
### Disable map
* enable / disable the map with TAB
### Render
* new file frame.c for the render
* removed some links between raycasts and the minimap
### TODO
* Scale the minimap

### CODE REVIEW

File: srcs/main.c [hunk 1]
```diff
@@ -17,6 +17,7 @@ int	main(int ac, char **av)
 	t_info	info;
 	int		i;
 
+	info.active_map = true;
```
Comment (t-h2o) [inline srcs/main.c]: Do we move this line into `init.c` ?

File: srcs/window.c [hunk 2]
```diff
@@ -6,16 +6,19 @@
...
 #include	"cub3d.h"
 
 // If the ESC key is pressed, the window closes
 static int
-	deal_key(int key, void *info)
+	deal_key(int key, void *params)
 {
+	t_info	*info;
+
+	info = params;
```
Comment (t-h2o) [inline srcs/window.c]: I already tried it, that work
> commit message if we commit this change: `window.c: cast the parameters to t_info`
```suggestion
	deal_key(int key, t_info *info)
{
```

**Review by t-h2o — COMMENTED** [review]
First review

File: srcs/main.c (hunk 1 above)
Comment (Giggzzz) [reply srcs/main.c]: I didn't see it was there. Yes we should move it

Comment (Giggzzz) [reply srcs/main.c]: c4ee0ee247e5b440abfff1bb716624da2478f02d

**Review by t-h2o — APPROVED** [review]
Good !

**Pull request merged**